bindings/pyroot_experimental/cppyy/CMakeLists.txt

```diff
@@ -1,4 +1,8 @@
-set(CMAKE_SHARED_LIBRARY_SUFFIX ".so")
+if(MSVC)
```

Comment:
This interplays nicely with this other condition check for the library extension, I guess:
https://github.com/root-project/root/blob/master/bindings/pyroot_experimental/cppyy/cppyy-backend/cling/python/cppyy_backend/loader.py#L13

Reply:
Well, ".so" file extension is not supported at all on Windows

Reply:
I am aware, I just wanted to make sure both condition checks will hold in the same cases.

Reply:
Oh, yes, indeed :slightly_smiling_face: